draft-fossati-tls-attestation.md

```diff
@@ -113,89 +114,37 @@ informative:
 
 --- abstract
 
-Attestation is the process by which an entity produces evidence about itself
-that another party can use to evaluate the trustworthiness of that entity.
-
-In use cases that require the use of remote attestation, such as confidential computing
-or device onboarding, an attester has to convey evidence or attestation results to 
-a relying party. This information exchange may happen at different layers in the 
-protocol stack. 
-
-This specification provides a generic way of passing evidence and attestation results
-in the TLS handshake. Functionality-wise this is accomplished with the help of key
-attestation.
+Attestation is the process by which an entity produces evidence about itself that another party can use to evaluate the trustworthiness of that entity.
+This document describes a series of protocol extensions to the TLS 1.3 handshake that enables the binding of the TLS authentication key to a remote attestation session.
```

Comment:
The binding to an attestation *session* is an artifact of the solution, because what we want to attest is a platform.

More generally, I think the new text is much more difficult to read than the old one (4 sentences that depend on one another in a logical sequence). Maybe the old text should be polished a little bit, but I still prefer it as a starting point.

Reply:
> The binding to an attestation session is an artifact of the solution, because what we want to attest is a platform.

I don't think of it as an emerging property.  It's a precise goal.

Reply:
I don't find it as difficult, but I think reshuffling a bit would make it nicer to read. For example, putting the current line 120 (about attestation giving you more comprehensive security metrics) immediately after L117.

For the TLS part (lines 118-119 at the moment) maybe it'd be easier to follow, particularly by people not familiar with these technologies, if we start by stating the goal of "attestation for authentication during setup of TLS channel".

Reply:
In the call we thought that, given all the tweaking that occurred in the last two weeks, this is now resolved.